testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -86,7 +79,7 @@
 import static java.util.stream.Collectors.toList;
 
 public class TestHiveStorageFormats
-        extends HiveProductTest
+        extends ProductTest
```

Comment:
OK to keep as `HiveProductTest`.

Reply:
I actually introduced it previously in https://github.com/trinodb/trino/pull/10309 for being able to get the hive version within `testInsertAllSupportedDataTypesWithTrino` .
I was thinking that since this is not needed anymore to revert the change.